packages/agoric-cli/README.md

```diff
@@ -1,56 +1,24 @@
-# Agoric Javascript Smart Contract Platform
+# Agoric CLI
 
-## Choose your Prerequisite
+If you just want to use the Agoric CLI for your own smart contract, please see the [Getting Started website](https://agoric.com/documentation/getting-started/) for information.
 
-<details><summary>Run directly from NPM using `npx`</summary>
-**NOTE: This doesn't work right now due to missing published packages.**
+## Developing Agoric CLI
 
-You will need a local installation of Node.js, at least version 11.
+**NOTE: these steps are only for modifying the Agoric CLI.  See the above for using it to create your own smart contracts.**
 
-```sh
-alias agoric='npx agoric'
-```
-</details>
-
-<details><summary>Install from NPM globally on your host</summary>
-**NOTE: This doesn't work right now due to missing published packages.**
-
-You will need a local installation of Node.js, at least version 11.
-
-```sh
-npm install -g agoric
-unalias agoric
-```
-</details>
-
-<details><summary>Use as part of Agoric SDK</summary>
-
-If you have cloned and installed the Agoric SDK as described by its [README](/Agoric/agoric-sdk/#readme), you can use the Agoric CLI directly.
+If you want to modify the `template` directory used by Agoric CLI, you can run:
 
 ```sh
-alias agoric="/PATH/TO/agoric-sdk/packages/agoric-cli/bin/agoric --sdk"
+# Change to the template directory
+cd template
+# Install any needed links
+../bin/agoric --sdk install
```

Comment:
What does this do? I get this error:

agoric: ERROR: [Error: EEXIST: file already exists, symlink '/Users/katelynsills/code/agoric-sdk/node_modules' -> 'node_modules'] {
  errno: -17,
  code: 'EEXIST',
  syscall: 'symlink',
  path: '/Users/katelynsills/code/agoric-sdk/node_modules',
  dest: 'node_modules'
}

Reply:
Mea culpa!  You shouldn't need this step.  I'll remove it.